.github/workflows/scale-test-egw.yaml

```diff
@@ -0,0 +1,458 @@
+        required: true
+      context-ref:
+        description: "Context in which the workflow runs. If PR is from a fork, will be the PR target branch (general case). If PR is NOT from a fork, will be the PR branch itself (this allows committers to test changes to workflows directly from PRs)."
+        required: true
+      SHA:
+        description: "SHA under test (head of the PR branch)."
+        required: true
+      extra-args:
+        description: "[JSON object] Arbitrary arguments passed from the trigger comment via regex capture group. Parse with 'fromJson(inputs.extra-args).argName' in workflow."
+        required: false
+        default: "{}"
+      num-clients:
+        description: "Number of clients to create to connect to the external target through EGW"
+        required: false
+        default: 100
+        type: number
+      client-qps:
+        description: "Number of client pods to create per second"
+        required: false
+        default: 20
+        type: number
+
+# For testing uncomment following lines:
+#  push:
+#    branches:
+#      - your_branch_name
+
+permissions:
+  # To be able to access the repository with actions/checkout
+  contents: read
+  # To be able to request the JWT from GitHub's OIDC provider
+  id-token: write
+  # To allow retrieving information from the PR API
+  pull-requests: read
+  # To be able to set commit status
+  statuses: write
+
+concurrency:
+  # Structure:
+  # - Workflow name
+  # - Event type
+  # - A unique identifier depending on event type:
+  #   - schedule: SHA
+  #   - workflow_dispatch: PR number
+  #
+  # This structure ensures a unique concurrency group name is generated for each
+  # type of testing, such that re-runs will cancel the previous run.
+  group: |
+    ${{ github.workflow }}
+    ${{ github.event_name }}
+    ${{
+      (github.event_name == 'schedule' && github.sha) ||
+      (github.event_name == 'workflow_dispatch' && github.event.inputs.PR-number)
+    }}
+  cancel-in-progress: true
+
+env:
+  # renovate: datasource=golang-version depName=go
+  go_version: 1.23.4
+  # renovate: datasource=github-releases depName=eksctl-io/eksctl
+  eksctl_version: v0.198.0
+  # renovate: datasource=github-releases depName=kubernetes/kubernetes
+  kubectl_version: v1.31.4
+  # renovate: datasource=docker depName=google/cloud-sdk
+  gcloud_version: 503.0.0
+
+  # Hosted under quay.io/cilium/egw-scale-utils and built by
+  # a workflow in cilium/scaffolding.
+  egw_utils_ref: 147e53d962f818eec901549c29be48b70b02b8d7
+  test_name: egw
+  cluster_name: ${{ github.run_id }}-${{ github.run_attempt }}
+  AWS_SSH_KEY: ~/.ssh/id_aws.pub
+  AWS_SSH_KEY_PRIVATE: ~/.ssh/id_aws
+
+jobs:
+  echo-inputs:
+    if: ${{ github.event_name == 'workflow_dispatch' }}
+    name: Echo Workflow Dispatch Inputs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Echo Workflow Dispatch Inputs
+        run: |
+          echo '${{ tojson(inputs) }}'
+
+  commit-status-start:
```

Comment:
The symmetric `status-final` job is missing.

Reply:
Thanks!